jasmine/spec/inverted-index-test.js

```diff
@@ -0,0 +1,88 @@
+describe('Index', function() {
+
+  var index = new Index();
+
+  describe('Read book data', function() {
+
+
+    it('should check if an Invalid Arguement is passed', function() {
+      expect(index.createIndex()).toBe('Invalid Arguement');
+    });
+
+    it('should check if the file passed is a valid file', function(done) {
+
+      index.createIndex('/test').catch(function(err) {
+        expect(err).toEqual(jasmine.any(Error));
+        expect(err.toString()).toEqual('Error: Not Found');
+        done();
+      });
+
+    });
+
+
+    it('should check if the file passed is a valid JSON and not empty', function(done) {
+
+      index.createIndex('/emptyBook.json').catch(function(err) {
+        expect(err).toEqual(jasmine.any(Error));
+        expect(err.toString()).toEqual('SyntaxError: Unexpected end of JSON input');
+        done();
+      });
+    });
+
+
+  });
+
+  describe('Populate Index', function() {
+    it('should assert that the Index is created after the json is read', function(done) {
+      expect(index.getIndex()).toBe('No Index Created');
+
+      index.createIndex('/books.json').then(function(result) {
+        expect(result).toBeTruthy();
+        done();
+      });
+    });
+
+
+    it('should read the JSON file and assert its not empty ', function(done) {
+      index.readJson('/books.json').then(function(response) {
+        expect(response).toBeTruthy();
+        done();
+      });
+    });
+
+
+  });
+
+  describe('Search index', function() {
+    beforeEach(function(done) {
+      index.createIndex('/books.json').then(function() {
+        done();
```

Comment:
**Code Quality - Nitpick**

Use a directory relative path (`books.json` or `./books.json`) for the same reason explained in this comment - https://github.com/andela-ekahindi/inverted-index/pull/1/files#r68094542